libs/platform/src/lib/components/form/radio-group/radio-group.component.ts

```diff
@@ -139,13 +144,10 @@ export class RadioGroupComponent extends CollectionBaseInput implements AfterVie
         if (!this._validateRadioButtons()) {
             throw new Error('fdp-radio-button-group must contain a fdp-radio-button');
         }
-
-        if (this.contentRadioButtons && this.contentRadioButtons.length > 0) {
-            this.contentRadioButtons.forEach((button) => {
-                this._selectUnselect(button);
-                this._changeDetector.detectChanges();
-            });
-        }
+        this.contentRadioButtons.forEach((button) => {
+            button.stateType = this.status;
+        });
```

Comment:
You can remove brackets and do it in 1 line

Reply:
Done